source/common/http/conn_manager_impl.cc

```diff
@@ -281,7 +281,7 @@ Network::FilterStatus ConnectionManagerImpl::onData(Buffer::Instance& data, bool
     const Status status = codec_->dispatch(data);
 
     ASSERT(!isPrematureResponseError(status));
-    if (isBufferFloodError(status)) {
+    if (isBufferFloodError(status) || isInboundFramesWithEmptyPayloadError(status)) {
```

Comment:
Would it be appropriate to ASSERT(status.ok()); on line 292, after the end of this if block?

Question is related to the ASSERT(!isPrematureResponseError(status)); on the previous line, is there a danger of us ignoring some of error status that is added in the future?  Granted, this seems like a pre-existing issue.

Reply:
Yes, I think it is a good suggestion.